src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/catchparametername/Example1.java

```diff
@@ -6,32 +6,37 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.naming.catchparametername;
+
 // xdoc section -- start
-public class MyTest {
+public class Example1 {
   public void myTest() {
     try {
       // ...
```

Comment:
Please replace all of these comments with an empty line, in all examples. They are just clutter and make the examples more difficult to read.

Reply:
fixed and pushed the changes.

Reply:
Not done, comment is gone, but there is no empty line. Please always allow reviewers to resolve their items so that they can double check the changes vs comments left.